modules/misc/qt.nix

```diff
@@ -3,6 +3,14 @@
 let
   cfg = config.qt;
 
+  platformPackages = with pkgs; {
```

Comment:
If you're going to drop `with`, I think you should drop all instances of it.

Reply:
In general `with` is discouraged at the top level but a scoped one like this usage is generally considered fine.

Also I think this is a by maintainer decision and I am the maintainer of this module, so unless there is a good reason I prefer to keep it as is.

Reply:
Sorry, that was just a suggestion.

>a scoped one like this usage is generally considered fine

It's more that it gets used like this *in spite* of the pitfalls, because it's convenient. Either way, I'm not here to bikeshed, so it's your call.